test/distributed/test_store.py

```diff
@@ -743,6 +744,31 @@ def test_wait(self, i):
 
 instantiate_parametrized_tests(TestMultiThreadedWait)
 
+class InitPgWithUvStore(TestCase):
+    def test_with_url_param(self):
+        port = common.find_free_port()
+        dist.init_process_group("gloo", rank=0, world_size=1, init_method=f"tcp://{DEFAULT_HOSTNAME}:{port}?use_libuv=1")
+        self._run_test()
+
+    def test_with_env_var(self):
+        port = common.find_free_port()
+        os.environ["USE_LIBUV"] = "1"
+        os.environ["MASTER_ADDR"] = DEFAULT_HOSTNAME
+        os.environ["MASTER_PORT"] = str(port)
```

Comment:
do we need to remove these env vars in `tearDown()` so they don't affect subsequent tests?

Reply:
fixing it.